package.json

```diff
@@ -22,16 +22,14 @@
     "lint": "esw -c .eslintrc *.js server config . --color --fix",
```

Comment:
This is still running the `--fix` before the push. Need to update to be on par with the other microservices.